docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,543 @@
+      control-center is [UP]
+
+
+#. Clone the |cp| `examples` repo in GitHub and work in the `clients/avro/` subdirectory.
+
+   .. sourcecode:: bash
+
+      $ git clone https://github.com/confluentinc/examples.git
+      $ cd examples/clients/avro
+   
+
+.. _schema_registry_tutorial_definition:
+
+Terminology
+^^^^^^^^^^^
+
+First let us levelset on terminology: what is a `topic` versus a `schema` versus a `subject`.
+
+A Kafka `topic` contains messages, and each message is a key-value pair.
+Either the message key or the message value, or both, can be serialized as Avro.
+A `schema` defines the structure of the Avro data format.
+The Kafka topic name can be independent of the schema name.
+By default, |sr| defines a scope in which schemas for that Kafka topic can evolve, and that scope is the `subject`.
+
+As a practical example, let's say a retail business is streaming transactions in a Kafka topic called `transactions`.
+A producer is writing data with a schema `Payment` to that Kafka topic.
+If the producer is serializing the message value as Avro, |sr| has a subject called `transactions-value`.
+If the producer is also serializing the message key as Avro, |sr| would have a subject called `transactions-key`, but for simplicity, in this tutorial you only consider the message value.
+The |sr| subject `transactions-value` has at least one schema called `Payment`.
+The subject `transactions-value` defines the scope in which schemas for the topic transactions can evolve and |sr| does compatibility checking within this scope.
+If developers evolve the schema `Payment` and produce new messages to the topic `transactions`, |sr| checks that those newly evolved schemas are compatible with older schemas in the subject `transactions-value` and adds those new schemas to the subject.
+
+.. _schema_registry_tutorial_definition:
+
+Schema Definition
+~~~~~~~~~~~~~~~~~
+
+The first thing developers need to do is agree on a basic schema for data.
+Client applications form a contract: producers will write data in a schema and consumers will be able to read that data.
+Of course, applications can use many schemas for many topics, but in this tutorial you will look at one.
+
+Consider the `original Payment schema <https://github.com/confluentinc/examples/blob/DEVX-380/clients/avro/src/main/resources/avro/io/confluent/examples/clients/basicavro/Payment.avsc>`_:
+
+.. sourcecode:: json
+
+   $ cat src/main/resources/avro/io/confluent/examples/clients/basicavro/Payment.avsc
+   {"namespace": "io.confluent.examples.clients.basicavro",
+    "type": "record",
+    "name": "Payment",
+    "fields": [
+        {"name": "id", "type": "string"},
+        {"name": "amount", "type": "double"}
+    ]
+   }
+
+Let's break down what this schema defines
+
+* ``namespace``: a fully qualified name that avoids schema naming conflicts
+* ``type``: `Avro data type <https://avro.apache.org/docs/1.8.1/spec.html#schemas>`_, one of `record`, `enum, `union`, `array`, `map`, `fixed`
```

Comment:
missing end \` on `enum`. Also, think these all should be double-ticked

Reply:
Ack